This mod alters the Calendar app to default to the month view.
Though it alters a function called launchDayView, re-pointing it at the
month scene, there are no known side effects under webOS 1.1.

Mod by matthewn, from http://forums.precentral.net/web-os-development/192542-howto-change-default-view-calendar-hack-rooting-required.html
Index: /usr/palm/applications/com.palm.app.calendar/app/controllers/app-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.calendar/app/controllers/app-assistant.js
+++ /usr/palm/applications/com.palm.app.calendar/app/controllers/app-assistant.js
@@ -126,12 +126,12 @@ var AppAssistant = Class.create({
 			// If the top scene is not the Day View, try to pop to it.
 			// If the top scene is already the Day View, there's no point
 			// in popping the scene
-			if (stageController.topScene().sceneName != "day") {
-				stageController.popScenesTo("day");
+			if (stageController.topScene().sceneName != "month") {
+				stageController.popScenesTo("month");
 			}
 			
 			// Check to see if the day scene is already up
-			if (stageController.topScene().sceneName == "day") {
+			if (stageController.topScene().sceneName == "month") {
 				sceneExists = true;
 			}
 			
@@ -152,7 +152,7 @@ var AppAssistant = Class.create({
 		
 		// Only push the Day View if it's not already visible
 		if (!sceneExists) {
-			stageController.pushScene({name: "day", transition: Mojo.Transition.crossFade, disableSceneScroller: true});
+			stageController.pushScene({name: "month", transition: Mojo.Transition.crossFade, disableSceneScroller: true});
 		}
 	},
 	
@@ -163,14 +163,14 @@ var AppAssistant = Class.create({
 		}
 		if (launchParams && launchParams.reminders) {
 			if (!appExists) {
-				stageController.pushScene({name: "day", disableSceneScroller: true});
+				stageController.pushScene({name: "month", disableSceneScroller: true});
 			}
 			stageController.pushScene("reminder-list");
 			return;
 		}
 		if (launchParams && launchParams.details) {
 			if (!appExists) {
-				stageController.pushScene({name: "day", disableSceneScroller: true});
+				stageController.pushScene({name: "month", disableSceneScroller: true});
 			}
 			// TODO need to pass event occurence date instead of today's date
 			stageController.pushScene('edit', launchParams.details, new Date(), false);
@@ -178,7 +178,7 @@ var AppAssistant = Class.create({
 		}
 		if (launchParams && launchParams.newEvent) {
 			if (!appExists) {
-				stageController.pushScene({name: "day", disableSceneScroller: true});
+				stageController.pushScene({name: "month", disableSceneScroller: true});
 			}
 			
 			this.createNewEventAndShowDetails(stageController, launchParams.newEvent);
@@ -357,7 +357,7 @@ var AppAssistant = Class.create({
 			this.firstUseManager = new FirstUseManager(stageController);
 		} else {
 			// Otherwise show the Day View
-			stageController.pushScene({name: "day", transition: Mojo.Transition.crossFade, disableSceneScroller: true});
+			stageController.pushScene({name: "month", transition: Mojo.Transition.crossFade, disableSceneScroller: true});
 		}
 	},
 	
